src/glyph/parse.rs

```diff
@@ -460,40 +438,31 @@ fn start(reader: &mut Reader<&[u8]>, buf: &mut Vec<u8>) -> Result<Glyph, Error>
                 let mut format: Option<GlifVersion> = None;
                 for attr in start.attributes() {
                     let attr = attr?;
-                    if attr.key == b"name" {
-                        name = attr.unescape_and_decode_value(&reader)?;
-                    } else if attr.key == b"format" {
-                        let value = attr.unescaped_value()?;
-                        let value = reader.decode(&value)?;
-                        format = Some(
-                            value
-                                .parse()
-                                .map_err(|e: ErrorKind| e.to_error(reader.buffer_position()))?,
-                        );
+                    // XXX: support `formatMinor`
```

Comment:
what does this mean?

Reply:
A new `formatMinor` attribute was added to the spec: https://unifiedfontobject.org/versions/ufo3/glyphs/glif/#glyph. SO we can have UFO v3.1 at some point.